tests/security/tpm2/tpm2_measured_boot.pm

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020-2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: TPM2 measured boot test
+#
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#108386
+
+use strict;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+
+sub run {
+    return unless get_var('QEMUTPM');
+
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Measured boot basic check, in current test logic, it depends on backend qemu and ovmf
+    # packages version to check measured boot on VM side. But we have other test modules to
+    # cover it on both TW and SLES
+    if (get_var('QEMUTPM') && get_var('QEMUTPM_VER') eq '2.0') {
```

Comment:
we don't need to check `QEMUTPM` again here. Have checked in above code.

Reply:
Agree with you